pyproject.toml

```diff
@@ -1,261 +1,266 @@
-[build-system]
-requires = ["setuptools>=61.2", "setuptools-scm"]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools.packages]
-find = {namespaces = false}
-
-[tool.setuptools.package-dir]
-"" = "."
+[tool.poetry]
+# required ones
+name = "geoips"         # geoips package available at the root of the project
+version = "0.0.0"
+description = "Geolocated Information Processing System"
+authors = ["Tom <tom.jerry@colostate.edu>"]
 
-[tool.setuptools.package-data]
-"*" = [
-    "plugins/*.txt",
-    "plugins/*/*.txt",
-    "plugins/*/*/*.txt",
-    "plugins/*/*/*/*.txt",
-    "plugins/*/*/*/*/*.txt",
-    "plugins/*.yaml",
-    "plugins/*/*.yaml",
-    "plugins/*/*/*.yaml",
-    "plugins/*/*/*/*.yaml",
-    "plugins/*/*/*/*/*.yaml",
-    "schema/*.yaml",
-    "schema/*/*.yaml",
-    "schema/*/*/*.yaml",
-    "schema/*/*/*/*.yaml",
-    "schema/*/*/*/*/*.yaml",
+# optional ones
+maintainers = ["Tom <tom.jerry@colostate.edu>"]
+#homepage = "set the url"   // optional
+repository = "https://github.com/NRLMMD-GEOIPS/geoips"    #optional
+#documentation = "set the url"      // optional
+#license = {file = "LICENSE"}   // required
+readme = "README.md"
+keywords = ["GeoIPS", "Poetry"]
+packages = [{include = "geoips"}]  # not needed as tool.poetry.name is properly set already
+classifiers = [                # // list of PyPI trove classifiers to describe the project
+    "Programming Language :: Python :: 3", ]
+include = [
+    "*.txt",
+    "*.py",
+    "*.yaml",
+    "yaml_configs/*",
+    "yaml_configs/*/*",
+    "yaml_configs/*/*/*",
     "image_utils/ascii_palettes/*",
 ]
 
-[tool.setuptools_scm]
-"version_scheme" = "post-release"  # Use current version .postN vs incrementing
-"local_scheme" = "no-local-version"  # Don't include hash, or date, just postN
-"write_to" = "geoips/_version.py"  # Write an actual file
+# uses current version .postN and not incrementing
+[tool.poetry-dynamic-versioning]
+enable = true
+vcs = "git"
+metadata = false    # don't include local-version hash; date doesn't appear by deafult
+style = "pep440"    # pep440 is deafult; can rmeove later on; used for : keeping 0.0.8 as the default install until 0.0.9 is out
+#style = "semver"
+#tag-branch = "feature-poetry"
+strict = true
+pattern = "(?x)^((?P<epoch>\\d+)!)?(?P<base>\\d+(\\.\\d+)*)([-._]?((?P<stage>[a-zA-Z]+)[-._]?(?P<revision>\\d+)?))?(\\+(?P<tagged_metadata>.+))?$"
+                         #           also: 3.6.0 incompatible with 0.20.3, but 0.21.0 works
```

Comment:
That looks like a cartopy comment.

Reply:
This was from matplolib; removed just now on the branch [feature-poetry](https://github.com/SRIKANTH-KUMAR/geoips/commit/bf80d793f664574860ef4e4e4f9c8ba0ca16530e).